lib/storefront/person2.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 39.15 35.15"><defs><style>.cls-1{fill:#b9b9b9;}</style></defs><title>Asset 2</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path class="cls-1" d="M12.67,9.75a5.38,5.38,0,1,0-6.23,0A9.57,9.57,0,0,0,1.07,23.21a.75.75,0,0,0,1.33-.69,8.07,8.07,0,0,1,.47-8.2L4.7,34.47a.75.75,0,0,0,.75.68h.07a.75.75,0,0,0,.68-.82l-2-21.56a8,8,0,0,1,5.08-2l.23,0,.25,0a8,8,0,0,1,5.12,2L13,34.33a.75.75,0,0,0,.68.82h.07a.75.75,0,0,0,.75-.68l1.84-20.15a8.07,8.07,0,0,1,.47,8.2.75.75,0,1,0,1.33.69A9.57,9.57,0,0,0,12.67,9.75ZM9.8,9.24H9.31a3.89,3.89,0,1,1,.49,0Z"/><path class="cls-1" d="M7.37,34.34a.74.74,0,0,0,.77.73.75.75,0,0,0,.73-.77L8.56,24.23h2l-.31,10.06a.75.75,0,0,0,.73.77h0a.75.75,0,0,0,.75-.73l.36-11.61H7Z"/><path class="cls-1" d="M32.67,9.75a5.38,5.38,0,1,0-6.23,0,9.57,9.57,0,0,0-5.37,13.44.75.75,0,0,0,1.33-.69,8.07,8.07,0,0,1,.47-8.2L24.7,34.47a.75.75,0,0,0,.75.68h.07a.75.75,0,0,0,.68-.82l-2-21.56a8,8,0,0,1,5.08-2l.23,0,.25,0a8,8,0,0,1,5.12,2L33,34.33a.75.75,0,0,0,.68.82h.07a.75.75,0,0,0,.75-.68l1.84-20.15a8.07,8.07,0,0,1,.47,8.2.75.75,0,0,0,1.33.69A9.57,9.57,0,0,0,32.67,9.75ZM29.8,9.24h-.49a3.89,3.89,0,1,1,.49,0Z"/><path class="cls-1" d="M27.37,34.34a.74.74,0,0,0,.77.73.75.75,0,0,0,.73-.77l-.31-10.06h2l-.31,10.06a.75.75,0,0,0,.73.77h0a.75.75,0,0,0,.75-.73l.36-11.61H27Z"/></g></g></svg>
```

Comment:
usually for these, we want to actually have no fill ie they should be `fill:none` on both these assets. we can then use `fill` on the css to fill in the colors

also we need to call it real things - they should have real `<title>` and `<desc>` tags describing what they are like `Two people`... oyuc an see what we do with other icons there